### DISCUSSION TITLE
Firefox's tracking protection domain is actually a tracker ??

### DISCUSSION THREAD

**Discussion by maxoux53** [badmojr/1Hosts#725] [Q&A] [answered]
Does someone know why this domain: ``tracking-protection.cdn.mozilla.net`` is blocked by several lists while it's basically used by the anti-trackink protection of Firefox ?
https://linksharing.samsungcloud.com/jt0ltKtyae97

**Comment by yoshimo** [comment, marked as answer]
Some people consider any connection they didn't explicitly gave their permission and were asked to opt-in as tracking and a violation of their privacy. Even if Mozilla is kind of trustworthy depending on your personal opinion
